contrib/src/main/java/com/datatorrent/contrib/hbase/AbstractHBaseWindowAppendOutputOperator.java

```diff
@@ -121,20 +98,12 @@ public void storeAggregate() {
   public abstract Append operationAppend(T t);
 
   @Override
-  public void processTuple(T tuple) {
-    tuples.add(tuple);
-  }
-
-  @Override
   public void setup(OperatorContext context)
   {
     mode=context.getValue(context.PROCESSING_MODE);
```

Comment:
context.PROCESSING_MODE => OperatorContext.PROCESSING_MODE

(to get rid of "The static field Context.OperatorContext.PROCESSING_MODE should be accessed in a static way")